app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -49,6 +51,16 @@
     Set<String> mutedTopics;
     private static final String MUTED_TOPIC_KEY = "mutedTopics";
 
+    private final static String GLOBAL_SETTINGS = "HumbugActivity";
+    private final static String SERVER_SETTINGS = "serverSettings";
+    private final static String GLOBAL_SETTINGS_CURRENT = "currentRealm";
+    private static final String GLOBAL_SETTINGS_REALMS = "REALMS";
+
+
+    public int currentRealm = 0;
+
+    public List<String> serverStringSet;
```

Comment:
Is there a reason this is a list and the data model that you're storing is a set?